public/style.css

```diff
@@ -2880,7 +2880,8 @@ select option:not(:checked) {
 }
 
 #instruct_enabled_label .menu_button:not(.toggleEnabled),
-#sysprompt_enabled_label .menu_button:not(.toggleEnabled) {
+#sysprompt_enabled_label .menu_button:not(.toggleEnabled),
+#send_banned_tokens_label .menu_button:not(.toggleEnabled) {
```

Comment:
I mean it's okay, but shouldn't we just start adding a CSS class to `menu_button` items that we want to show in red when they are not `.toggleEnabled`?
Or like, maybe just make it generic. Why would you not want a not toggle-enabled button to not be red?

The third button here might mean there'll be more in the future, so could be a nice abstraction.